docker/build_docker.sh

```diff
@@ -289,6 +290,8 @@ function docker_build {
     if docker_login; then
         env DOCKER_CONTENT_TRUST=$docker_trust DOCKER_CONFIG="${DOCKER_CONFIG}"  docker push ${image_full_name}
         echo "Done docker push for: ${image_full_name}"
+        PUSHED_DOCKERS="${image_full_name},$PUSHED_DOCKERS"
```

Comment:
There will be an extra comma at the end. Is that ok?

Reply:
yeah it actually gets parsed with `sed 's/,$//')` in the end to drop the trailing comma downstream. Let me know if thats too ugly =)

Reply:
No, it's ok. Leave it as is.